addon/components/uni-header-menu.js

```diff
@@ -0,0 +1,20 @@
+import Ember from 'ember';
+import layout from '../templates/components/uni-header-menu';
+import headerButtons from '../enums/header-buttons';
+import headerOptions from '../enums/header-menu';
+import languages from '../enums/languages';
+
+const { Component } = Ember;
+
+export default Component.extend({
+  headerButtons: headerButtons.values(),
+  headerOptions: headerOptions.values(),
+  languageOptions: languages.values(),
+  exitLabel: 'Exit',
+  tagName: '',
+  layout,
+
+  actions: {
+    triggerExit() {}
```

Comment:
This way you won't be able to implement the function, as the actions: {} is local to the component.

```js 
do 

triggerExit() {},

actions: {
    triggerExit() {
         this.get('triggerExit')();
    }
}

```